src/fidesops/schemas/saas/shared_schemas.py

```diff
@@ -22,6 +22,7 @@ class SaaSRequestParams(BaseModel):
     path: str
     params: Dict[str, Any]
     body: Optional[str]
+    ignore_errors: Optional[bool] = False
```

Comment:
Just an opinion, I wonder if `ignore_errors` should be included as part of the SaaSRequestParams. I always thought of this class as being the way to track the different components of an HTTP request. Some fields like path and params get modified after going through the pagination logic, but ignore_errors just gets forwarded along. What if we pass this value in directly when we call the send function? ```self.client().send(prepared_request, saas_request.ignore_errors)```. That way we don't have to worry about it in the pagination logic, or any future code that uses SaaSRequestParams.

I approve of the current implementation and we don't have to make any changes now but I just wanted to know your thoughts.

Reply:
thanks @galvana, updated to just pass this through via SaasRequest - end-to-end testing to be completed in https://github.com/ethyca/fidesops/issues/256